src/example-types/facet.js

```diff
@@ -2,6 +2,19 @@ let F = require('futil')
 let _ = require('lodash/fp')
 let { ObjectID } = require('mongodb')
 
+let projectStageFromLabelFields = node => {
+  let labelFields = _.get('valueLabelFields', node)
+  return {
+    $project: {
+      count: 1,
+      ..._.zipObject(
```

Comment:
maybe shorten to `F.arrayToObject(fieldName => `labelData.${fieldName}`, _.constant(1))(labelFields)`